client/src/tpu_client.rs

```diff
@@ -55,27 +62,12 @@ impl TpuClient {
         self.tpu_client.try_send_wire_transaction(wire_transaction)
     }
 
-    /// Create a new client that disconnects when dropped
-    pub fn new(
-        rpc_client: Arc<RpcClient>,
-        websocket_url: &str,
-        config: TpuClientConfig,
-    ) -> Result<Self> {
-        let connection_cache = ConnectionCache::default();
-        Self::new_with_connection_cache(
-            rpc_client,
-            websocket_url,
-            config,
-            Arc::new(connection_cache.into()),
-        )
-    }
-
```

Comment:
As CI is pointing out, this is a breaking change to a public interface. We'll have to find some other way.

Reply:
https://github.com/solana-labs/solana/pull/30128#discussion_r1098747493